src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__stars {
+    width: 16px;
+    height: 16px;
+    background-position: center;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+}
+
+.stars--4 .stars__stars:nth-child(-n + 4) {
```

Comment:
All modifiers should work properly, consider adding styles for all classes with modifiers from the previous task

Reply:
not fixed 
![Uploading image.png…]()